components/layout/Header/LangToggle/index.tsx

```diff
@@ -38,11 +38,11 @@ function LangToggle(props: LangToggleProps) {
 
   return (
     <Select onValueChange={onChangeLocale}>
-      <SelectTrigger className="w-auto min-w-[6.25rem]">
+      <SelectTrigger className="w-auto min-w-[6.25rem] text-[12px] md:text-[16px]">
```

Comment:
```suggestion
      <SelectTrigger className="w-auto min-w-[6.25rem] text-[0.75rem] md:text-[1rem]">
```
Mà 2 cái này có text-3 với text-4 thì phải, check lại xem